src/core/MOM_density_integrals.F90

```diff
@@ -1739,8 +1738,7 @@ real function frac_dp_at_pos(T_t, T_b, S_t, S_b, z_t, z_b, rho_ref, G_e, pos, EO
   real,           intent(in)  :: G_e !< The Earth's gravitational acceleration [L2 Z-1 T-2 ~> m s-2]
   real,           intent(in)  :: pos !< The fractional vertical position, 0 to 1 [nondim]
   type(EOS_type), intent(in)  :: EOS !< Equation of state structure
-  real                        :: fract_dp_at_pos !< The change in pressure from the layer top to
```

Comment:
This description includes the dimensions of the output variable, which should be moved up to the description of the function itself.